sdk/eventhub/event-hubs/src/eventDataBatch.ts

```diff
@@ -37,57 +36,56 @@ export interface TryAddOptions {
 }
 
 /**
- * A interface representing a batch of events which can be passed to the `send` method of a `EventProducer` instance.
- * This batch is ensured to be under the maximum message size supported by Azure Event Hubs service.
- *
- * Use `createBatch()` method on the `EventHubProducer` to create an instance of `EventDataBatch`
- * instead of using `new EventDataBatch()`. You can specify an upper limit for the size of the batch
- * via options when calling `createBatch()`.
- *
- * Use the `tryAdd` function on the EventDataBatch to add events to the batch. This method will return
- * `false` after the upper limit is reached, therefore check the result before calling `tryAdd()` again.
+ * An interface representing a batch of events which can be used to send events to Event Hub.
+ * 
+ * To create the batch, use the `createBatch()` method on the `EventHubProducerClient`.
+ * To send the batch, use the `sendBatch()` method on the same client.
+ * To fill the batch, use the `tryAdd()` method on the batch itself.
+ * 
  */
 export interface EventDataBatch {
   /**
-   * The partitionKey set during `EventDataBatch` creation. This value is hashed to
-   * produce a partition assignment when the producer is created without a `partitionId`
-   * If this value is set then partitionId can not be set.
+   * A value that is hashed and used by the Azure Event Hubs service to determine the partition to
```

Comment:
Just to clarify - this is the _name_ of the key that contains the value or is it the value itself?

Reply:
The value itself